chia/full_node/full_node_api.py

```diff
@@ -1453,15 +1453,13 @@ async def request_block_headers(self, request: wallet_protocol.RequestBlockHeade
 
         if request.end_height < request.start_height or request.end_height - request.start_height > 128:
             return make_msg(ProtocolMessageTypes.reject_block_headers, reject)
-        if self.full_node.block_store.db_wrapper.db_version == 2:
```

Comment:
We now always try the new `get_block_bytes_in_range` API and only fall back to the version 1 api if we get a `NotImplementedError` exception